src/temporal/client/worker.clj

```diff
@@ -55,22 +55,30 @@ Options for configuring the worker-factory (See [[start]])
   "
 Options for configuring workers (See [[start]])
 
-| Value                                          | Mandatory   | Description                                                       | Type             | Default |
-| ------------                                   | ----------- | ----------------------------------------------------------------- | ---------------- | ------- |
-| :task-queue                                    | y           | The name of the task-queue for this worker instance to listen on. | String / keyword |         |
-| :ctx                                           |             | An opaque handle that is passed back as the first argument of [[temporal.workflow/defworkflow]] and [[temporal.activity/defactivity]], useful for passing state such as database or network connections.  | <any> | nil |
-| :dispatch                                      |             | An optional map explicitly setting the dispatch table             | See below        | All visible activities/workers are automatically registered |
-| :max-concurrent-activity-task-pollers          |             | Number of simultaneous poll requests on activity task queue. Consider incrementing if the worker is not throttled due to `MaxActivitiesPerSecond` or `MaxConcurrentActivityExecutionSize` options and still cannot keep up with the request rate. | int | 5 |
-| :max-concurrent-activity-execution-size        |             | Maximum number of activities executed in parallel.                | int              | 200     |
-| :max-concurrent-local-activity-execution-size  |             | Maximum number of local activities executed in parallel.          | int              | 200     |
-| :max-concurrent-workflow-task-pollers          |             | Number of simultaneous poll requests on workflow task queue.      | int              | 2       |
```

Comment:
the default used to be 2, but is [now 5](https://javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/worker/WorkerOptions.Builder.html#setMaxConcurrentWorkflowTaskPollers(int))